changelog.txt

```diff
@@ -3,6 +3,7 @@
 = 1.25.8 - 2021-xx-xx =
 * Tweak - Display better errors on checkout page when address fields are missing / invalid.
 * Fix   - UPS invoice number allows numbers and letters.
+* Add   - Update the existing endpoint `POST /connect/packages` to create shipping label packages, and add an endpoint `PUT /connect/packages` to update shipping label packages.
```

Comment:
Yea it'd be ideal if this breaking change can be in a new version so that it doesn't affect any users of this endpoint (Chris mentioned Calypso is one of them other than the WCS web client and it's being removed, but maybe there are external developers using this endpoint).

If API versioning is implemented soon (this or next release), we can certainly wait for the endpoint to be versioned 😄 For Woo mobile, we also have another new endpoint on shipping label creation eligibility https://github.com/Automattic/woocommerce-services/pull/2334 that we'd love to release in the near future that is higher priority than this package creation endpoint.

Reply:
We had a brief discussion on slack (p1614723041112800-slack-CU6SYV31A). We think the risk/impact is low and this is okay to go ahead

We talked about:

**Affecting areas**
- Only impact users that has custom code on the "shipping label creation UI" or the "admin create package UI". We don't have a metric on that but we don't think there would be a lot of users.
- Affect adding packages. 

**Impact**
- ~~They get duplicated packages~~ They won't be able to update packages.

**What user should do when upgrading**
Similar to how we upgrade npm packages., we expect users to look at the changelog when they upgrade.

Reply:
Thank you for confirming this with the team! In case of any existing endpoint users, getting an error sounds better than getting duplicated packages 🙂